admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,179 @@
+.. OPcache is enabled by default and ``phpinfo`` displays status and statistics.
+..
+.. .. figure:: images/cache-1.png
+..   :alt: The Zend OPcache section displays opcode cache status and statistics.
+..  
+..   *Figure 1: Zend OPcache status in phpinfo*
+..   
+.. If it is not enabled, then go into    
+
+Memcached
+---------
+
+Memcached is a reliable oldtimer for shared caching on distributed servers, 
+and performs well with ownCloud with one exception: it is not suitable to use 
+with :doc:`Transactional File Locking <../configuration_files/files_locking_transactional>` because it does not 
+store locks, and data can disappear from the cache at any time (Redis is 
+the best for this). 
+
+Setting up Memcached is easy. On Debian/Ubuntu/Mint install ``memcached`` and 
+``php5-memcached``. The installer will automatically start ``memcached`` and 
+configure it to launch at startup.
+
+On Red Hat/CentOS/Fedora install ``memcached`` and 
+``php-pecl-memcached``. It will not start automatically, so you must use 
+your service manager to start ``memcached``, and to launch it at boot as a 
+daemon.
+ 
+You can verify that the Memcached daemon is running with ``ps ax``::
+
+ ps ax | grep memcached
+ 19563 ? Sl 0:02 /usr/bin/memcached -m 64 -p 11211 -u memcache -l 
+ 127.0.0.1
+
+Then add the appropriate entries to your ``config.php``. This example uses APCu 
+for the local cache, Memcached as the distributed memcache, and lists all the 
+servers in the shared cache pool with their port numbers::
+
+ 'memcache.local' => '\OC\Memcache\Memcached',
+ 'memcache.distributed' => '\OC\Memcache\Memcached',
+ 'memcached_servers' => array(
+      array('localhost', 11211),
+      array('server1.example.com', 11211),
+      array('server2.example.com', 11211), 
+      ), 
+
+Redis
+-----
+
+Redis is an excellent modern memcache to use for both distributed caching, and 
+with :doc:`Transactional File Locking <../configuration_files/files_locking_transactional>` because it guarantees 
+that cached objects are available for as long as they are needed.
+
+The Redis PHP module must be version 2.2.5 or better.
+
+On Debian/Ubuntu/Mint install ``redis-server`` and ``php5-redis``. The installer 
+will automatically launch ``redis-server`` and configure it to launch at 
+startup.
+
+On Red Hat/CentOS/Fedora install ``redis`` and ``php-pecl-redis``. It will not 
+start automatically, so you must use your service manager to start 
+``memcached``, and to launch it at boot as a daemon.
+ 
+You can verify that the Redis daemon is running with ``ps ax``::
+ 
+ ps ax | grep redis
+ 22203 ? Ssl    0:00 /usr/bin/redis-server 127.0.0.1:6379 
+ 
+This example ``config.php`` configuration uses APCu for the local server cache, 
+and sets up three servers in the shared cache pool:: 
+
+ 'memcache.local' => '\OC\Memcache\APCu',
+ 'memcache.distributed' => '\OC\Memcache\Redis',     
+ 'redis' => array(
+	'host' => 'localhost', 
```

Comment:
I'm not sure of this Redis config. I need a minimal working configuration to start, and can expand the Redis section over time as I learn more about it.